test/functional/options/defaults_spec.lua

```diff
@@ -328,3 +329,64 @@ describe('XDG-based defaults', function()
     end)
   end)
 end)
+
+
+describe('stdpath()', function()
+  before_each(function()
+    clear({env={
+      XDG_CONFIG_HOME='cat',
+      XDG_CONFIG_DIRS='siamese:persian',
+      XDG_DATA_HOME='dog',
+      XDG_DATA_DIRS='chihuahua:terrier',
+      XDG_CACHE_HOME='capybara',
+    }})
+  end)
+
+  it('knows the config home', function()
+    eq('cat/nvim', eval('stdpath("config")'))
+  end)
+
+  it('knows the data home', function()
+    if os_name() == 'windows' then
+      eq('dog\\nvim-data', eval('stdpath("data")'))
```

Comment:
@docwhat Yes; yes; at the start of the smallest scope possible; just import `funcs` from `helpers`, indexing will work automatically.